tests/contrib/test_gce.py

```diff
@@ -66,11 +66,13 @@ def _refresh_success_helper(self, bytes_response=False):
         credentials.refresh(http)
         self.assertEquals(access_token, credentials.access_token)
 
-        base_metadata_uri = ('http://metadata.google.internal/0.1/meta-data/'
-                             'service-accounts/default/acquire')
+        base_metadata_uri = (
+            'http://metadata.google.internal/computeMetadata/v1/instance/'
+            'service-accounts/default/acquire')
         escaped_scopes = urllib.parse.quote(' '.join(scopes), safe='')
         request_uri = base_metadata_uri + '?scope=' + escaped_scopes
-        http.request.assert_called_once_with(request_uri)
+        headers = { 'Metadata-Flavor': 'Google' }
```

Comment:
As was done in the non-test file: drop the space to the right of `{` and to the left of `}` and elide the local "headers" field.